azext_edge/edge/command_map.py

```diff
@@ -204,17 +204,35 @@ def load_iotops_commands(self, _):
     with self.command_group(
         "iot ops connector opcua trust",
         command_type=connector_resource_ops,
+        is_preview=True,
     ) as cmd_group:
         cmd_group.command("add", "add_connector_opcua_trust")
+        cmd_group.command("remove", "remove_connector_opcua_trust")
+        cmd_group.show_command("show", "show_connector_opcua_trust")
 
     with self.command_group(
         "iot ops connector opcua issuer",
         command_type=connector_resource_ops,
+        is_preview=True,
     ) as cmd_group:
         cmd_group.command("add", "add_connector_opcua_issuer")
+        cmd_group.command("remove", "remove_connector_opcua_issuer")
+        cmd_group.show_command("show", "show_connector_opcua_issuer")
 
     with self.command_group(
         "iot ops connector opcua client",
         command_type=connector_resource_ops,
+        is_preview=True,
     ) as cmd_group:
         cmd_group.command("add", "add_connector_opcua_client")
+        cmd_group.command("remove", "remove_connector_opcua_client")
+        cmd_group.show_command("show", "show_connector_opcua_client")
+
+    with self.command_group(
+        "iot ops schema version",
```

Comment:
This schema version command group is duplicate, it is defined above connector

Reply:
got it, let me double check, probably got this accidently during resolving conflict